hoodie-client/src/main/java/com/uber/hoodie/config/HoodieIndexConfig.java

```diff
@@ -47,6 +47,15 @@
   public static final String DEFAULT_BLOOM_INDEX_USE_CACHING = "true";
   public static final String BLOOM_INDEX_TREE_BASED_FILTER_PROP = "hoodie.bloom.index.use.treebased.filter";
   public static final String DEFAULT_BLOOM_INDEX_TREE_BASED_FILTER = "true";
+  // On by default. Once stable, we will remove the other mode.
+  public static final String BLOOM_INDEX_BUCKETIZED_CHECKING_PROP = "hoodie.bloom.index.bucketized.checking";
```

Comment:
s/BLOOM_INDEX_BUCKETIZED_CHECKING_PROP/BLOOM_INDEX_BUCKETIZED_CHECK_PROP
rename "hoodie.bloom.index.bucketized.checking" to "hoodie.bloom.index.bucketized.check"

Reply:
Can we add the comments above as a TODO since it warrants removal of something which is easier to find with TODO ?

Reply:
added TODO.. like to keep it checking, since there are more than 1 check ..